tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/TestHello.java

```diff
@@ -94,6 +103,26 @@ public void testHello() throws TornadoExecutionPlanException {
         }
     }
 
+    @Test
+    public void testPrintIntArray() throws TornadoExecutionPlanException {
+        assertNotBackend(TornadoVMBackendType.SPIRV);
```

Comment:
Please add also the `assertNotBackend` check for the PTX type.

Reply:
Yes, good catch